modules/ding_content/ding_content.features.wysiwyg.inc

```diff
@@ -51,20 +54,18 @@ function ding_content_wysiwyg_default_profiles() {
           'media' => 1,
         ),
       ),
-      'toolbar_loc' => 'top',
-      'toolbar_align' => 'left',
-      'path_loc' => 'bottom',
-      'resizing' => 1,
-      'verify_html' => 1,
-      'preformatted' => 0,
-      'convert_fonts_to_spans' => 1,
-      'remove_linebreaks' => 1,
-      'apply_source_formatting' => 0,
-      'paste_auto_cleanup_on_paste' => 0,
-      'block_formats' => 'p,address,pre,h2,h3,h4,h5,h6,div',
+      'toolbarLocation' => 'top',
+      'resize_enabled' => 1,
+      'default_toolbar_grouping' => 0,
+      'simple_source_formatting' => 0,
+      'acf_mode' => 0,
+      'acf_allowed_content' => '',
       'css_setting' => 'self',
       'css_path' => '/profiles/ding2/themes/ddbasic/css/ddbasic.wysiwyg.css',
-      'css_classes' => '',
+      'stylesSet' => '',
+      'block_formats' => 'p,address,pre,h2,h3,h4,h5,h6,div',
```

Comment:
Supporting all these formats seems excessive compared to the original issue. Besides making things more complex I also fear that all these headings have not been styled properly.

How about reducing this to `p,h2,h3,div`